content/source/docs/enterprise/saml/identity-provider-configuration-aad.html.md

```diff
@@ -69,7 +133,24 @@ Follow these steps to configure Azure Active Directory (AAD) as the identity pro
     ```
 
     ![Screenshot: New role in Manifest Editor](./images/sso-aad-saml-manifest-devapprole.png)
-5. Go back to "Enterprise applications", and select the app you created for TFE. In the left sidebar, under the "Manage" heading, select "Users and Groups". This is where you will enable access to TFE by adding either users or groups to your application. During the process of adding users or groups you will select a role to be assigned to the user or group. Select the role that matches the user or groups TFE team.
+
+5. Go back to **Enterprise applications**, and select the app you created for TFE. In the left sidebar, under the **Manage** heading, select **Users and Groups**. This is where you will enable access to TFE by adding either users or groups to your application. During the process of adding users or groups you will select a role to be assigned to the user or group. Select the role that matches the user or groups TFE team.
+
     ![Screenshot: Role Assignment](./images/sso-aad-saml-role-assignment.png)
 
 Once users have been added, the initial configuration is complete, and they can begin logging into TFE with their AAD username and password.
+
+
+## Test SSO
+
+In this section, you test your Azure AD single sign-on configuration with following options. 
+
+* Click on **Test this application** in Azure portal. This will redirect to Terraform Enterprise Sign-on URL where you can initiate the login flow. 
+
+* Go to Terraform Enterprise Sign-on URL directly and initiate the login flow from there.
+
```

Comment:
```suggestion
- Click on **Test this application** in Azure portal. This will redirect to Terraform Enterprise Sign-on URL where you can initiate the login flow. 
- Go to Terraform Enterprise Sign-on URL directly and initiate the login flow from there.
```